src/token.rs

```diff
@@ -56,6 +102,7 @@ pub enum SgfToken {
     Time { color: Color, time: u32 },
     PlayerName { color: Color, name: String },
     PlayerRank { color: Color, rank: String },
+    RU(Rule),
```

Comment:
Think the name here should be `RuleSet` as well

Reply:
So RuleSet(RuleSet) ?

Reply:
Hmm.. When you mention it, it might look better as `SgfToken::Rule(RuleSet::AGA)`, rather then `SgfToken::RuleSet(RuleSet::Aga)`. Feels a bit redundant to use `RuleSet` both places.